### PR TITLE
fix custom namespace

### PR DESCRIPTION
**Pull request by arturobrzut**
Move RBAC for Operand from Yaml file in bundle folder inside CSV like we have it in 0.X operator
It works 
but of course not supported
but also RBAC inside bundle format is not supported and we can not create Operand RBAC in a different namespace (namespace we have to hardcoded)
the supported scenario is to add it to operator logic and create an inside operator but this will require cluster permission to create ClusterRoleBinding 
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/46443
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/46418

### CODE REVIEW

**Comment by arturobrzut** [comment]
@pgodowski comment is needed to merge it

**Comment by arturobrzut** [comment]
https://github.com/operator-framework/operator-lifecycle-manager/issues/2063#issuecomment-822650782

**Comment by arturobrzut** [comment]
image
image

File: Makefile [hunk 1]
```diff
@@ -463,6 +463,22 @@ alm-example:
...
 	`yq r -P  -j ./config/samples/operator.ibm.com_v1alpha1_ibmlicensingrequest.yaml`\
 	]"
+	yq r -P ./bundle/manifests/ibm-license-service_rbac.authorization.k8s.io_v1_clusterrole.yaml rules > /tmp/clusterrole.yaml
+	yq r -P ./bundle/manifests/ibm-license-service_rbac.authorization.k8s.io_v1_role.yaml rules > /tmp/role.yaml
+	sed -i -e 's/^/  /' /tmp/clusterrole.yaml
+	sed -i -e 's/^/  /' /tmp/role.yaml
+	cp ./common/scripts/updateCSV/updateCP.yaml /tmp/updateCP.yaml
+	cat /tmp/clusterrole.yaml >> /tmp/updateCP.yaml
+	cat ./common/scripts/updateCSV/saCP.yaml >> /tmp/updateCP.yaml
+	yq w -i -s /tmp/updateCP.yaml ./bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml
```
Comment (pgodowski) [inline Makefile:473]: given we are automatically adding entries into CSV, please also add commented out line into CSV explainign WHY each of the cluster permissions are required.
See https://github.com/IBM/ibm-management-ingress-operator/blob/master/bundle/manifests/ibm-management-ingress-operator.clusterserviceversion.yaml#L264


Unless, once CSV is imported, commnents are discarded on the actual cluster?

**Review by pgodowski — APPROVED** [review]
approved conditionally, granted you answer my question. I am ok that the optional CSV commnts can be added into subsequent PR

File: common/scripts/updateCSV/saP.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- command: update
```
Comment (pgodowski) [inline common/scripts/updateCSV/saP.yaml:1]: please change the filenames, the file `saP.yaml` is meaningless. I am ok for you to do this in separate PR

**Review by pgodowski — APPROVED** [review]
